04-HTML5-APIs/Ex03/script.js

```diff
@@ -0,0 +1,42 @@
+let output = document.getElementById("output");
+document.body.onload = document.getElementById('disconnect').disabled = true;
+
+var websocket;
+
+function wsConnect(event) {
+  let wsUri = "ws://echo.websocket.org/";
+  websocket = new WebSocket(wsUri);
```

Comment:
You can skip this comment, I use to separate the sections in the code, that way is more readable

```
let weConnect = function (event) {
    let wsUri = ....
    //blank line because var declaration finish 
    websocket = ....
};
```